server/middleware/token.js

```diff
@@ -32,6 +32,9 @@ function rewriteUserLiteral(req, currentUserLiteral) {
       debug('req.url has been rewritten from %s to %s', urlBeforeRewrite,
         req.url);
     }
+  } else if (!req.accessToken && currentUserLiteral &&
+      new RegExp(currentUserLiteral).test(req.url)) {
+    req.res.status(401).end();
```

Comment:
ok, I wasn't sure if it was better to `throw` or `return`, prob picked the wrong one but this is updated 👍